apps/sms/js/dialog.js

```diff
@@ -21,6 +21,7 @@
     id: 'localizableStringWithArgument',
     args: { n: count }
   },
+  classes: ['specific-class'], // optional
```

Comment:
Just curious that if this  `classes` is optional, why don't we move it into `options` below?